### PR TITLE
Fix flaky Kubernetes tests

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Review by mkouba — APPROVED** [review]

File: integration-tests/kubernetes/standard/src/test/java/io/quarkus/it/kubernetes/BasicKubernetesTest.java [hunk 1]
```diff
@@ -40,9 +39,9 @@
     private Path logfile;
 
     @Test
-    public void asertApplicationRuns() throws IOException {
+    public void asertApplicationRuns() {
```
Comment (machi1990) [inline integration-tests/kubernetes/standard/src/test/java/io/quarkus/it/kubernetes/BasicKubernetesTest.java]:
```suggestion
    public void assertApplicationRuns() {
```

File: integration-tests/kubernetes/standard/src/test/java/io/quarkus/it/kubernetes/KubernetesWithHealthTest.java [hunk 2]
```diff
@@ -44,10 +43,9 @@
     private Path logfile;
 
     @Test
-    public void asertApplicationRuns() throws IOException {
+    public void asertApplicationRuns() {
```
Comment (machi1990) [inline integration-tests/kubernetes/standard/src/test/java/io/quarkus/it/kubernetes/KubernetesWithHealthTest.java]:
```suggestion
    public void assertApplicationRuns() {
```

File: integration-tests/kubernetes/standard/src/test/java/io/quarkus/it/kubernetes/WithKubernetesClientTest.java [hunk 3]
```diff
@@ -42,10 +41,9 @@
     private Path logfile;
 
     @Test
-    public void asertApplicationRuns() throws IOException {
+    public void asertApplicationRuns() {
```
Comment (machi1990) [inline integration-tests/kubernetes/standard/src/test/java/io/quarkus/it/kubernetes/WithKubernetesClientTest.java]:
```suggestion
    public void assertApplicationRuns() {
```

**Review by machi1990 — DISMISSED** [review]
Spotted a typo. Otherwise looks good to me.

**Comment by geoand** [comment]
@machi1990 good catch indeed!

However the typo was introduced by a previous PR and I really want to get this fix in ASAP because there are CI failures showing up in PRs.

Feel free to open a new PR to fix this typo and I'll approve it immediately :)

**Comment by geoand** [comment]
I ended up applying the changes @machi1990 because there was some totally irrelevant failure so I rebased onto master

**Review by machi1990 — APPROVED** [review]

**Comment by machi1990** [comment]
> I ended up applying the changes @machi1990 because there was some totally irrelevant failure so I rebased onto master

Thanks!

**Pull request merged**